crates/extensions/tedge_mqtt_ext/src/lib.rs

```diff
@@ -121,7 +122,8 @@ impl MqttMessageBox {
     }
 
     async fn send(&mut self, message: MqttMessage) -> Result<(), ChannelError> {
-        self.log_output(&message.clone());
+        log_message_sent("MQTT", &message);
```

Comment:
MqttMessageBox does not have a logging sender because the current version logs any message you sending even if they're filtered by the topics and not sent to a peer.